CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 ## CHANGE LOG
 
+### Version 2.2.0 *(19 March 2024)*
+-------------------------------------------
+**What's new**
+* **[Android Platform]**
+  * Supports [CleverTap Android SDK v6.1.1](https://github.com/CleverTap/clevertap-android-sdk/blob/master/docs/CTCORECHANGELOG.md#version-611-february-27-2024).
+
+* **[iOS Platform]**
+  * Supports [CleverTap iOS SDK v6.1.0](https://github.com/CleverTap/clevertap-ios-sdk/releases/tag/6.1.0).
+  * Adds privacy manifests.
+
+* **[Android and iOS Platform]**
+  * Fixes an issue of incorrect endpoint in the case of network handshake. 
+  * Fixes a bug in Client Side InApps with regards to frequency limits.
+
+**Bug Fixes**
+* **[Android Platform]**
+  * Fixes InApps crash in a rare activity destroyed race condition 
+  * Fixes Potential ANR in a race condition of SDK initialisation in multithreaded setup 
+  * Fixes #456 - Build issues due to AGP 8
```

Comment:
This point was native specific, can be removed

Reply:
done